### ISSUE TITLE
Possible to not use arrays ? can be closed 

### ISSUE THREAD

**Issue by lanaiscool** [aceat64/EasyBitcoin-PHP#20]
possible to not use an array for params so it send params: {} instead of params:[]

**Comment by aceat64** [comment]
Why?

**Comment by lanaiscool** [comment]
let me rephrase that possible to use named params instead of positional ? because my rpc server does return invalid request when the json string uses params:[] and it works fine when i create a json string with params:{} i dont know if this is a difficult job to change ?

**Comment by aceat64** [comment]
What version are you running? An empty array should be valid, and works fine for me against v0.15.1.

**Comment by lanaiscool** [comment]
i am using a customized version when i delete the params => $params in easybitcoin.php and request a method that doesnt require any parameters it works fine

**Comment by lanaiscool** [comment]
YOu can close this comment as i fixed it by adding JSON_FORCE_OBJECT
